powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -393,6 +393,39 @@ void testIdentityVerificationCleanup() throws Exception {
         powerAuthClient.removeActivation(activationId, "test");
     }
 
+    @Test
+    void testIdentityVerificationMaxAttemptLimit() throws Exception {
+        String[] context = prepareActivation();
+        String activationId = context[0];
+        String processId = context[1];
+
+        approveConsent(processId);
+        for (int i = 0; i < 5; i++) {
+            initIdentityVerification(activationId, processId);
+
+            List<FileSubmit> idDocSubmits = ImmutableList.of(
+                    FileSubmit.createFrom("images/id_card_mock_front.png", DocumentType.ID_CARD, CardSide.FRONT),
+                    FileSubmit.createFrom("images/id_card_mock_back.png", DocumentType.ID_CARD, CardSide.BACK)
+            );
+            DocumentSubmitRequest idCardSubmitRequest = createDocumentSubmitRequest(processId, idDocSubmits);
+            submitDocuments(idCardSubmitRequest, idDocSubmits);
+
+            if (i < 4) {
+                cleanupIdentityVerification(processId);
+            } else {
+                try {
```

Comment:
This could be maybe after the preparation phase and be covered by
```
Exception exception = assertThrows(OnboardingException.class, () -> {
   cleanupIdentityVerification(processId);
});
```